tests/test_likelihood.py

```diff
@@ -1,86 +1,77 @@
-            cls.disease_to_phenotype_file,
-            )
-        cls.phenotype_groups = read_phenotype_groups()
-
-    def test_read_phenotype_groups_1000(self):
-        hp_to_pg = read_phenotype_groups()
-        self.assertEqual(hp_to_pg['HP:0012759']['k1000'], 62)
-
-    def test_read_phenotype_groups_1500(self):
-        hp_to_pg = read_phenotype_groups()
-        self.assertEqual(hp_to_pg['HP:0012759']['k1500'], 739)
-
-    def test_predict_likelihood(self):
-        phenotypes = [
-            ['HP:0012759', 'HP:0003011', 'HP:0011442'], 
-            ['HP:0012759', 'HP:0003011'],
-        ]
-        probabilities = predict_likelihood_moldx(phenotypes, self.phenotype_groups, self.hpo_network, self.alt2prim)
-        self.assertAlmostEqual(probabilities[0], 0.33, places=2)
-
-    def test_predict_likelihood_phenotypes_only(self):
-        phenotypes = [
-            ['HP:0012759', 'HP:0003011', 'HP:0011442'], 
-            ['HP:0012759', 'HP:0003011'],
-        ]
-        probabilities = predict_likelihood_moldx(phenotypes)
-        self.assertAlmostEqual(probabilities[0], 0.33, places=2)
-
-    def test_no_hpo_config_section(self):
-        config.remove_section('hpo')
-        phenotypes = [
-            ['HP:0012759', 'HP:0003011', 'HP:0011442'], 
-            ['HP:0012759', 'HP:0003011'],
-        ]
-        with self.assertRaises(NoSectionError):
-            predict_likelihood_moldx(phenotypes)
-
-    def test_no_hpo_config_option(self):
-        config.remove_option('hpo', 'disease_to_phenotype_file')
-        phenotypes = [
-            ['HP:0012759', 'HP:0003011', 'HP:0011442'], 
-            ['HP:0012759', 'HP:0003011'],
-        ]
-        with self.assertRaises(NoOptionError):
-            predict_likelihood_moldx(phenotypes)   
-
-    def test_bad_k(self):
-        phenotypes = [
-            ['HP:0012759', 'HP:0003011', 'HP:0011442'], 
-            ['HP:0012759', 'HP:0003011'],
-        ]
-        with self.assertRaises(KeyError):
-            predict_likelihood_moldx(
-                phenotypes, 
-                self.phenotype_groups,
-                self.hpo_network,
-                self.alt2prim, 
-                k_phenotype_groups=500
-                )
\ No newline at end of file
+def test_read_phenotype_groups_1000(test_data):
+    hp_to_pg = read_phenotype_groups()
+    assert hp_to_pg["HP:0012759"]["k1000"] == 62
+
+
+def test_read_phenotype_groups_1500(test_data):
+    hp_to_pg = read_phenotype_groups()
+    assert hp_to_pg["HP:0012759"]["k1500"] == 739
+
+
+def test_predict_likelihood(test_data):
+    phenotypes = [
+        ["HP:0012759", "HP:0003011", "HP:0011442"],
+        ["HP:0012759", "HP:0003011"],
+    ]
+    probabilities = predict_likelihood_moldx(
+        phenotypes,
+        test_data["phenotype_groups"],
+        test_data["hpo_network"],
+        test_data["alt2prim"],
+    )
+    assert round(probabilities[0], 2) == 0.33
+
+
+def test_predict_likelihood_phenotypes_only(test_data):
```

Comment:
I get a similar error as above on this test

Reply:
tests for `likelihood.py` will be removed as well